seaborn/tests/test_axisgrid.py

```diff
@@ -303,6 +303,18 @@ def test_legend_options(self):
         g1.map(plt.plot, "x", "y")
         g1.add_legend()
 
+        g1 = ag.FacetGrid(self.df, hue="b")
+        g1.add_legend(adjust_subtitles=True)
+
+        g1 = ag.FacetGrid(self.df, hue="b")
+        g1.add_legend(adjust_subtitles=False)
```

Comment:
I think you can drop these two smoke tests, since these codepaths are hit many times by tests that cover the figure-level functions with defaults.

Reply:
Thanks, done!